docs/main/05-reference/02-resource-requests/01-deleting-a-resource-request.md

```diff
@@ -0,0 +1,28 @@
+---
+description: Deleting a Resource Request
+title: Deleting
+---
+
+To delete a Resource Request, run:
```

Comment:
Can this also be deleted by file? If so, I think we should be consistent with the delete a Promise file where we choose to list that as an option.